test/e2e/checkbox.test.js

```diff
@@ -10,49 +10,49 @@ describe('vl-checkbox', async () => {
     });
 
     it('als gebruiker kan ik een standaard checkbox aan- en uitvinken', async () => {
-        const checkbox1 = await vlCheckboxPage.getCheckbox(1);
-        await kanCheckboxAanEnUitvinken(checkbox1);
+        const checkbox = await vlCheckboxPage.getDefaultCheckbox(1);
```

Comment:
waarom hier dan nog de index (1) nodig?

Reply:
Omdat je checkbox 1 of 2 kan opvragen, specifiekere id's gaan voorzien heeft echt geen nut in kader van deze testen.

Reply:
dan zou ik nog getDefaultCheckbox in uw page-object maken die dan doordelegeert naar getDefaultCheckbox(1)